packages/runtime/container-runtime/src/summarizerHeuristics.ts

```diff
@@ -56,7 +62,8 @@ export class SummarizeHeuristicRunner implements ISummarizeHeuristicRunner {
         private readonly heuristicData: ISummarizeHeuristicData,
         private readonly configuration: ISummaryConfiguration,
         private readonly trySummarize: (reason: SummarizeReason) => void,
-        private readonly minOpsForAttemptOnClose = 50,
+        private readonly logger: ITelemetryLogger,
+        private readonly minOpsForAttemptOnClose = 10,
```

Comment:
>10, [](http://example.com/codeflow?start=51&length=3)

We might wait for my PR so this default will not be here anymore - I could change on my PR but for history purposes it might be better for us to change later.